packages/sdk-codegen-utils/README.md

```diff
@@ -2,6 +2,11 @@
 
 This package contains utility routines for the SDK code generator projects
 
-The code in this package is really trivial and probably not pulling in as a dependency to anything.
+The code in this package is really trivial and should not be used directly in your own projects.
+
+This package has no node dependencies.
+
+**DEPRECATED AND NOT SUPPORTED**: Looker does not support direct use of this package, other than via direct use in the
```

Comment:
other than via direct use in the
should be?
other than via indirect use in the